### ISSUE TITLE
Use empty value for repository image alt text in "In person" sidebar

### ISSUE THREAD

**Issue by ggeisler** [projectblacklight/arclight#525]
For the image we show in the In person sidebar as part of the repository info:

<img width="267" alt="Screen Shot 2019-07-19 at 3 44 49 PM" src="https://user-images.githubusercontent.com/101482/61569526-85bc9880-aa3c-11e9-8688-1c2403b3f1a4.png">

we currently include an `alt` value with the repository name (i.e., we use `repository.name` to populate the `alt` attribute value). However, since immediately adjacent to that image we display a link with the repository name (populated using the same `repository.name` value), the screen reader user will hear the exact same repository name repeated twice in a row.

So in this context I believe we can consider the image to be decorative and thus not deserving of an `alt` attribute value. It should instead just be an empty value:`<img alt="" class="img-fluid float-left" src="https:` etc.

**Comment by jkeck** [comment]
Closed by #527
